### ISSUE TITLE
zoom limited

### ISSUE THREAD

**Issue by heinkurz** [der-stefan/OpenTopoMap#347]
Hi!
First thanks for OpenTopoMap, it's great.

My question:
Did you limit the zoom level a view weeks ago? In the left lower edge I can see a scale of 500 m. In the past I could go down to 200 m.

Thanks

**Comment by k1teflight** [comment]
Probably max zoom level limited to 14 as written in this post still applies: [https://github.com/der-stefan/OpenTopoMap/issues/346#issuecomment-1694775075](https://github.com/der-stefan/OpenTopoMap/issues/346#issuecomment-1694775075)

**Comment by der-stefan** [comment]
True, the renderer has been shut down. It is now server only pre-rendered tiles, until the new vector tiles are released.
